orchestra/context_processors.py

```diff
@@ -0,0 +1,10 @@
+from django.conf import settings
+
+
+def global_settings(request):
```

Comment:
yeah, I'm just worried that this will get abused in the future... Is there any way to associate a context_processor with a single template or a list of templates? Or is there relatively minor syntax to let us call something like `render('path/to/template', extra_context_processors=[global_settings,])`?